region/norfair/west/Crumble Shaft.json

```diff
@@ -395,25 +402,28 @@
       ],
       "note": "Shoot the shot block, then Spring Ball on the middle platform."
     },
-    {
-      "id": 11,
-      "link": [2, 1],
-      "name": "HiJump Climb",
-      "requires": [
-        "HiJump",
-        "canConsecutiveWalljump",
-        {"heatFrames": 500}
-      ],
-      "note": "Walljump up the right, then setup a walljump on the top-right crumble platform directly from the right wall."
-    },
     {
       "id": 12,
       "link": [2, 1],
-      "name": "HiJumpless Climb",
+      "name": "Wall Jump Climb",
       "requires": [
         "canPreciseWalljump",
         "canConsecutiveWalljump",
-        {"heatFrames": 500}
+        {"or": [
+          {"and": [
+            "HiJump",
+            {"heatFrames": 270}
+          ]},
+          {"heatFrames": 340}
```

Comment:
I can't get under 6 seconds unless I enter with a spinjump

Reply:
I was entering with a spinjump from a water room to simulate the worst-case scenario. We don't have an entrance condition to specifically represent this (issue https://github.com/vg-json-data/sm-json-data/issues/1788) though I know we sometimes use `canPrepareForNextRoom`. In this case, since it only saves a few heat frames, which can be accounted for in the multiplier, I was thinking it's better not to clutter up the requirements. Added a mention in the `notes` now though, because it's useful to document the expectations to be able to reproduce the heat frames.